packages/app/cypress/support/index.ts

```diff
@@ -23,10 +23,7 @@ Cypress.Commands.add('loginAsGuest', () => {
 });
 
 Cypress.Commands.add('saveGithubToken', () => {
-cy.visit('/', {
-    onLoad: (win: Window) =>
-     win.localStorage.setItem('githubSession', Cypress.env('githubSession')),
-  });
+  cy.intercept('GET', 'http://localhost:7007/api/auth/github/refresh?optional&env=development', { fixture: 'githubLogin/login.json' })
```

Comment:
Why this change was required?

Reply:
With some updates they made the github oauth app and github app more similar, and as I was testing locally with a github OAuth app it did not use the localstorage anymore to persist the session. So it meant on all pages where we use github plugins it propted to login and the cypress tests failed. 
The new way is that the frontend calls the backend to get the session, so if we intercept it here and return a valid format, it means we can trick the plugins that we are actually logged in so they can render the correct widgets without prompting to login.